python/ray/autoscaler/_private/event_summarizer.py

```diff
@@ -1,5 +1,6 @@
 from typing import Any, Callable, Dict, List
```

Comment:
Nothing changed in this file besides sticking methods behind a lock for thread safety.